bases/node-ts.json

```diff
@@ -4,6 +4,10 @@
   "docs": "https://devblogs.microsoft.com/typescript/announcing-typescript-5-8-beta/#the---erasablesyntaxonly-option",
   "_version": "23.6.0",
   "compilerOptions": {
+    "target": "esnext",
```

Comment:
This file was supposed to be a mixin placed at the end of an extends list: the TS extension thing seems good but I'm not sure about overriding the target or module.